stephenPrintup/README.md

```diff
@@ -0,0 +1,347 @@
+
+review:
+var w = r-l,
+    h = t-b,
+    diamSquared = h*h + w*w,
+    radiusSquared = diamSquared/4,
+    area = Math.PI * radiusSquared;
+
+
+**e)**
+Imagine subdividing your rectangle into 3 equal rows and 3 equal columns, which would create 9 smaller rectangles, identical in shape but varying by position.
+Define four new variables describing the centermost small rectangle.
+(_Hint: one of the many solutions is very similar to the solution of **2b** above._)
+
+answer: 
+var lc = (2*l + r)/3;
+var rc = (2*r + l)/3;
+var tc = (2*t + b)/3;
+var bc = (2*b + t)/3;
+
+
+---
+
+**4)** (_25%_)
+
+Imagine that the squares of an ordinary checkerboard are numbered in two different ways:
+
+* Each square has integer coordinates _(R,C)_ describing its row and column.  Both values should be in the range 0..7, so that the upper-left square is at (0,0) and the bottom-right is at (7,7).
+
+RC = "Integer Coordinates"
+R = "row integer"
+C = "column integer"
+RC = (R, C)
+
+* Each square has a unique integer number N from 0 to 63.  These numbers run sequentially left-to-right one row at a time, top to bottom.  Therefore the upper-left square has N===0 and the bottom-right has N===63.
+
+N = "Integer # 0 to 63"
+
+
+**a)**  Given a particular R and C, find the corresponding N.  That is, write an expression for variable N containing variables R and C.
+
+N = C + (R * 8) 
+
+**b)**  Given N, find R.  Write an expression for R which contains N.
+
+R = Math.floor(N / 8)
+
+**c)**  Given N, find C.  Write an expression for C which contains N.
+
+C = N % 8
+
+**d)**  Assume the squares are colored black and white, with the upper-left square black.
+Write an expression (or pair of conditional statements) to set a variable _color_ to either 'black' or 'white', describing the square identified by variables R,C, and N (or a subset of them, if you don't need all three).
+
+var color = black
+if (Math.even(N)) {
+	color = white;
+	} else { color = black;};
+
+answer:
+if (r%2 == c%2) {
+    var color = 'black';
+} else {
+    var color = 'white';
+}
+
+---
+
+**5)** (_15%_)
+
+Suppose you represent a fraction (_n/d_) with 2 integer variables: _n_ for the numerator and _d_ for the denominator.
+If _n_ is greater than _d_, the fraction is "improper", but it can be rewritten as a proper fraction.  For example, "7/4" is improper, but it can be rewritten as "1 3/4", which is proper.
+
+Assuming variables _n_ and _d_ are defined in advance (but you don't know their values), write a series of expressions to generate a string expressing the proper form of the fraction _n/d_.  For example, when _n===7_ and _d===4_, your resulting string should be "1 3/4".  You may assume both _n_ and _d_ are positive integers and _n_ > _d_, but otherwise you should be able to handle any values of _n_ and _d_.
+
+n = numerator
+d = denominator
+
+a) Solve it first by making use of a function called _Math.floor_.
+
+print(Math.floor(n/d) + " " + n%d + "/" + d)
+answer:
```

Comment:
JavaScript doesn't have any built in print functions, but I think what you were looking for was console.log (which is similar to the print function). 

Minus some syntax stuff, your answer is correct!